c++/triqs/mc_tools/mc_measure_set.hpp

```diff
@@ -154,8 +155,8 @@ namespace triqs {
 
       std::string report() const {
         std::string result{};
-        for (auto &nmp : m_map) {
-          result += " " + nmp.first + "=" + nmp.second.report();
+        for (auto &[name, m] : m_map) {
+          if (m.report_measure) result += m.report() + "\n";
```

Comment:
Should the measure print its own name or should the registered name be used? In the latter case this change is necessary.
```suggestion
          if (m.report_measure) result += name + "=" + m.report() + "\n";
```

Reply:
I would leave it up to the measurement how to format its report string and what to include. This gives us more flexibility.